src/qml/WiFiPage.qml

```diff
@@ -0,0 +1,149 @@
+            width: root.width
+            height: wifiStatus.powered ? width*0.6 : root.height
+            Behavior on height { NumberAnimation { duration: 100 } }
+            anchors.horizontalCenter: parent.horizontalCenter
+            Rectangle {
+                id: statusIconBackground
+                anchors.centerIn: parent
+                anchors.verticalCenterOffset: -parent.width*0.13
+                color: "black"
+                radius: width/2
+                opacity: wifiStatus.powered ? 0.4 : 0.2
+                width: parent.width*0.25
+                height: width
+                Icon {
+                    id: statusIcon
+                    anchors.fill: statusIconBackground
+                    anchors.margins: parent.width*0.12
+                    name: wifiStatus.powered ? "ios-wifi" : "ios-wifi-outline"
+                }
+                MouseArea {
+                    id: statusMA
+                    enabled: true
+                    anchors.fill: parent
+                    onClicked: wifiStatus.powered = !wifiStatus.powered
+                }
+            }
+
+
+            Label {
+                id: statusLabel
+                font.pixelSize: parent.width*0.07
+                horizontalAlignment: Text.AlignHCenter
+                verticalAlignment: Text.AlignVCenter
+                wrapMode: Text.Wrap
+                anchors.left: parent.left; anchors.right: parent.right
+                anchors.leftMargin: parent.width*0.04; anchors.rightMargin: anchors.leftMargin
+                anchors.verticalCenter: parent.verticalCenter
+                anchors.verticalCenterOffset: parent.width*0.15
+                text: "<h3>" + (wifiStatus.powered ? qsTrId("id-wifi-on"): qsTrId("id-wifi-off")) + "</h3>\n" + (wifiStatus.connected ? qsTrId("id-wifi-connected") : qsTrId("id-wifi-disconnected"))
+            }
+        }
+
+        footer: Item {height: wifiStatus.powered ? root.height*0.15 : 0; width: parent.width}
+
+        delegate: MouseArea {
+            property var wifiName: modelData.name
+            visible: wifiStatus.powered
+            width: wifiList.width
+            height: wifiStatus.powered ? width*0.23 : 0
+            Marquee {
+                id: wifiNameLabel
+                text: wifiName
+                height: parent.height*0.6
+                width: parent.width
+            }
+            Label {
+                anchors.top: wifiNameLabel.bottom
+                anchors.horizontalCenter: parent.horizontalCenter
+                opacity: 0.8
+                font.pixelSize: parent.width*0.07
+                font.weight: Font.Thin
+                text: {
+                    if (modelData.connected) {
+                        qsTrId("id-wifi-connected")
+                    } else if (modelData.favorite){
+                        qsTrId("id-wifi-saved")
+                    } else {
+                        qsTrId("id-wifi-notsetup")
```

Comment:
I believe you'd need to provide the string in a specially formatted comment just above these qsTrId

Reply:
Also a final PR would need to update the translation files, but you can keep this for when that's undrafted